package/CHANGELOG

```diff
@@ -53,6 +53,8 @@ Changes
     duplicates. Otherwise, the group itself is returned. Repeated access of
     *Group.unique will always return the same object unless the group is
     updated or modified. (PR #1922)
+  * SurvivalProbability does not dilute SP anymore when the reference frame (t0)
+    cannot find any molecules in the first place. (PR #1995)
```

Comment:
Can you also add both of your names to the author list contributing to this release at the top. Thanks.

Reply:
I assme you mean just above the  0.18.1 in CHANGELOG. Done

Reply:
Yes just above the 0.18.1. I'm not sure if you are done yet. Github doesn't show the new names. We tend to use github names for that list.